### ISSUE TITLE
[Question][Hws] Could you upload the test cases after due date of  each hw?

### ISSUE THREAD

**Issue by Allgot** [prosyslab-classroom/cs492-program-reasoning#65]
Hello.

I want to know which functionalities I've missed so far in homeworks. Could you release the test cases of each homework, after its due date? It would be very helpful for studying and understanding the materials.

Thank you!

**Comment by minseongg** [comment]
It seems like the hidden test cases are uploaded in the GitHub classroom repository.

I recommend you to check it!

**Comment by KihongHeo** [comment]
Yes. All the evaluation test cases must be already there. If not, ask TAs.

**Comment by Allgot** [comment]
Thanks a lot!
I've just checked, and they are there well :)
